lib/fluent/plugin/in_kafka_group.rb

```diff
@@ -264,6 +264,8 @@ def run
               when :record
                 if @time_format
                   record_time = @time_parser.parse(record[@record_time_key].to_s)
+                elsif record[@record_time_key].is_a?(Float)
+                  record_time = Fluent::NumericTimeParser.new(:float).parse(record[@record_time_key])
```

Comment:
Create NumberTimeParser in each record is extremely slow.
The parser should be created in `configure`.
In addition, call `record[@record_time_key]` twice is redundant. Following code is better

```
if @time_format
  # same as
else
  record_time = record[@record_time_key]
  if record_time.is_a?(Float)
    record_time = @float_numeric_parser.parse(record_time)
  end
end
```